ambry-store/src/test/java/com.github.ambry.store/PersistentIndexTest.java

```diff
@@ -899,7 +899,168 @@ public void testExistsWithFileSpan() throws Exception {
   @Test
   public void testFindEntries() throws Exception {
     // provide empty token and ensure we get everything till max
-    StoreFindToken token = new StoreFindToken();
```

Comment:
could you add a description to this test? (I know it is not new, but since you make all these changes, might as well make this up-to-date).